### PR TITLE
:sound: Ensure error messages are consistent

### PR DESCRIPTION
**Pull request by neilbmclaughlin**
Also changed page title to ensure context is given to screen readers

### CODE REVIEW

**Comment by neilbmclaughlin** [comment]
Just spotted that the html contains tabs - will correct and force push

**Comment by neilbmclaughlin** [comment]
Done and ready for review.

@mcheung-nhs @ermlikeyeah wondering if the `<div class="reading-width">` should be in layout.nunjucks since it is replicated in both error pages although find-help.nunjucks doesn't use it. Any thoughts?

**Comment by st3v3nhunt** [comment]
@neilbmclaughlin Some conflicts need sorting out please

File: app/views/error-404.nunjucks [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 {% block header %}
-Page not found  
+  {{ message }}
 {% endblock %}
 
 {% block content %}
   <div class="reading-width">
-		<p>If you have entered a web address check it was correct. You can <a href="//nhs.uk/">browse from the NHS.UK home page</a>.</p>
-		<p><a href="javascript:history.go(-1)" class="link-back">Back</a></p>
+    <p>If you have entered a web address check it was correct. You can <a href="//nhs.uk/">browse from the NHS.UK home page</a>.</p>
```
Comment (st3v3nhunt) [inline app/views/error-404.nunjucks]: The href here needs to be `http://www.nhs.uk` given the current site doesn't work over `https`

**Pull request merged**